packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/record-position.listener.ts

```diff
@@ -21,27 +20,31 @@ export class RecordPositionListener {
 
   @OnEvent('*.created')
   async handleAllCreate(payload: ObjectRecordCreateEvent<any>) {
-    if (!hasPositionField(payload.createdObjectMetadata)) {
+    if (!hasPositionField(payload.objectMetadata)) {
```

Comment:
See below, the reason there are a lot of renaming is that there was not standard convention for events related to entities creation/update/delete, it was getting messy as we were not always passing the same prop depending on context. I tried to improve this a bit. I feel like it's better, but could still be improved in a future iteration